### ISSUE TITLE
Provide similar API across Platforms

### ISSUE THREAD

**Issue by leotrs** [ManimCommunity/ManimPango#16]
Right now, the usage in `manim/mobject/svg/text_mobject.py` is this:

```py
        if sys.platform.startswith("linux"):
            manimpango.unregister_font()
        else:
            manimpango.unregister_font(str(file_path))
```

It would be nice to pull this check into `unregister_file` itself.

**Comment by naveen521kk** [comment]
Two things:
Either:
1. https://github.com/ManimCommunity/manim/pull/939 should be waited for this to be implemented and it can be part of v0.1.8 release.
2. Or merge that PR, and later add this and tag it v0.2.0 so that things don't break.
What do you think @leotrs? What can be done?

**Comment by leotrs** [comment]
Number 2 sounds like it breaks fewer things so let's go with that.
